packages/Search/cmake/Dependencies.cmake

```diff
@@ -4,4 +4,6 @@ TRIBITS_PACKAGE_DEFINE_DEPENDENCIES(
   Kokkos
   Teuchos
   Tpetra
+  LIB_REQUIRED_TPLS
```

Comment:
This one is a trickier one.  This is only use in the tests/examples.  So it could be optional with the tests only added if it is present.